pubtools/_quay/command_executor.py

```diff
@@ -335,20 +349,18 @@ def _run_cmd(self, cmd, err_msg=None, tolerate_err=False, stdin=None):
         """
         err_msg = err_msg or "An error has occured when executing a command."
         cmd_exec = self.client.exec_create(self.container["Id"], cmd)
-        stdout, stderr = self.client.exec_start(cmd_exec["Id"], demux=True)
+        # Unfortunately, older versions of Docker API don't support demuxing stdout and stderr
```

Comment:
Shouldn't the result depend on the version?

Reply:
The main issue is that older versions of the Python dependency didn't even support this option.